e2e-app/src/app/popover/autoclose/popover-autoclose.pw-spec.ts

```diff
@@ -0,0 +1,122 @@
+import {sendKey, openUrl, Key} from '../../tools.pw-po';
+import {test} from '../../../../playwright.conf';
+
+const SELECTOR_OPEN_STATUS = '#open-status';
+const SELECTOR_POPOVER = 'ngb-popover-window';
+
+const clickOutside = async() => await test.page.click('#outside-button');
+const rightClickOutside = async() => await test.page.click('#outside-button', {button: 'right'});
+
+const clickInside = async() => await test.page.click('div.popover-body');
+const rightClickInside = async() => await test.page.click('div.popover-body', {button: 'right'});
+
+const selectAutoClose = async(type: string) => {
+  await test.page.click('#autoclose-dropdown');
+  await test.page.click(`#autoclose-${type}`);
+};
+
+const expectPopoverToBeOpen = async(message: string) => {
+  await test.page.waitForSelector(SELECTOR_POPOVER)
```

Comment:
Missing semicolon.

Reply:
Fixed, thanks. I've added this `waitForSelector` in the attempt to fix webkit failing tests and looks like it works.